Resources/public/js/views/ez-usermenuitemfireeventview.js

```diff
@@ -42,6 +44,27 @@ YUI.add('ez-usermenuitemfireeventview', function (Y) {
         },
 
         /**
+         * Adds the event listener
+         *
+         * @method _addEventListener
+         * @protected
+         * @param event {Object} event facade
+         */
+        _addEventListener: function (event) {
```

Comment:
that's not a very descriptive name. maybe `_addUserMenuHideOnEvent` or something like that.